linode_api4/objects/linode.py

```diff
@@ -997,20 +1283,54 @@ def clone(
         self,
         to_linode=None,
         region=None,
-        service=None,
+        type=None,
```

Comment:
`type` is a reserved key work for a built-in function in Python
https://docs.python.org/3/library/functions.html#type

Reply:
The original name of the parameter seems to be a typo. The API documentation does not show any parameters called `service` for this endpoint. I believe the parameter is actually meant to represent the type of the instance. I'll rename it to `instance_type` or something like that so that I don't use a python keyword. Good catch!